app/serializers/v1/simple_intervention_serializer.rb

```diff
@@ -2,7 +2,7 @@
 
 class V1::SimpleInterventionSerializer < V1Serializer
   include TeamCollaboratorsHelper
-  attributes :id, :user_id, :name, :status, :created_at, :updated_at, :organization_id, :google_language_id
+  attributes :id, :user_id, :name, :status, :reports_deleted, :data_cleared, :created_at, :updated_at, :organization_id, :google_language_id
```

Comment:
Are you sure we need this in the simple serializer? I don't think so because this serializer contains only the necessary data to correctly display sessions on the main screen

Reply:
I thought we also wanted to show this data on the main dashboard, like an icon or something?

Reply:
Why? Do you have the design to prove it?

Reply:
I believe we were talking about this in a meeting, but if you think it's redundant I can remove it from this serializer